tgui/packages/tgui/styles/interfaces/AtmosHandbook.scss

```diff
@@ -0,0 +1,7 @@
+.AtmosHandbook__tooltip-content{
+	border-bottom: dotted 2px;
```

Comment:
style doesn't like it when I do it but if your SCSS is this small consider just using `style` 😶